.cirrus.yml

```diff
@@ -48,72 +48,121 @@ freebsd_environment: &FREEBSD_ENVIRONMENT
     ZEEK_CI_CPUS: 8
     ZEEK_CI_BTEST_JOBS: 8
 
-builds_only_if_template: &BUILDS_ONLY_IF_TEMPLATE
-  # Rules for skipping builds:
-  # - Do not run builds for anything that's cron triggered
-  # - Don't do darwin builds on zeek-security repo because they use up a ton of compute credits.
-  # - Always build PRs, but not if they come from dependabot
-  # - Always build master and release/* builds from the main repo
+builds_only_if_pr_master_release: &BUILDS_ONLY_IF_PR_MASTER_RELEASE
+  only_if: >
+    ( ( $CIRRUS_REPO_NAME == 'zeek' || $CIRRUS_REPO_NAME == 'zeek-security' ) &&
+      ( $CIRRUS_CRON != 'weekly' ) &&
+      ( $CIRRUS_PR != '' ||
+        $CIRRUS_BRANCH == 'topic/timw/ci-test-master' ||
+        $CIRRUS_BRANCH =~ 'topic/timw/ci-test-release/.*'
+      )
+    )
+
+builds_only_if_pr_master_release_nightly: &BUILDS_ONLY_IF_PR_MASTER_RELEASE_NIGHTLY
   only_if: >
-    ( $CIRRUS_CRON == '' ) &&
-    ( ( $CIRRUS_PR != '' && $CIRRUS_BRANCH !=~ 'dependabot/.*' ) ||
     ( ( $CIRRUS_REPO_NAME == 'zeek' || $CIRRUS_REPO_NAME == 'zeek-security' ) &&
-      (
-      $CIRRUS_BRANCH == 'master' ||
-      $CIRRUS_BRANCH =~ 'release/.*'
+      ( $CIRRUS_CRON != 'weekly' ) &&
+      ( $CIRRUS_PR != '' ||
+        $CIRRUS_BRANCH == 'topic/timw/ci-test-master' ||
+        $CIRRUS_BRANCH =~ 'topic/timw/ci-test-release/.*' ||
+        ( $CIRRUS_CRON == 'nightly' && $CIRRUS_BRANCH == 'topic/timw/ci-test-master' )
```

Comment:
Only staring, is the nightly check here redundant, given `$CIRRUS_BRANCH == 'topic/timw/ci-test-master' ||` further up?)
```suggestion
```

I like that `nightly` is there for explicitness, but it doesn't seem it makes a difference for the execution. But probably better to just keep it, so ignore the suggestion.